.gitignore

```diff
@@ -1,2 +1,157 @@
-/target
+
+# Created by https://www.toptal.com/developers/gitignore/api/rust,clion+all,emacs
+# Edit at https://www.toptal.com/developers/gitignore?templates=rust,clion+all,emacs
+
+### CLion+all ###
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio, WebStorm and Rider
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
+# User-specific stuff
+.idea/**/workspace.xml
```

Comment:
Well, actually, I would encourage you to add that to your own gitignore! https://stackoverflow.com/questions/5724455/can-i-make-a-user-specific-gitignore-file